index.html

```diff
@@ -38,5 +38,106 @@ <h1>I'm Shimels, Glad to see you!</h1>
         <a href="#"> <img src="icones/Vector.png" alt="" /></a>
       </section>
     </main>
+
+    <main class="project-container">
+      <div class="card">
+        <img src="images/bk-image.jpg" alt="project image" />
+        <h2>Tonic</h2>
```

Comment:
- [ ] Kindly stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). The text color of the title is different from the Figma design, kindly use the same color to show your ability to parse a Figma design to UI.